requirements.txt

```diff
@@ -1,8 +1,10 @@
 batinfo==0.3
+git+https://github.com/bochecha/dbus-python.git@pipified-1.1.1#egg=dbus
```

Comment:
Can you add here the comment you made on the first review on why we use your fork?
Also, can you move the fork on ideascube organization?